circuitbox.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency 'bundler', '~> 1.16'
   spec.add_development_dependency 'excon', '~> 0.62'
-  spec.add_development_dependency 'faraday', '~> 0.15'
+  spec.add_development_dependency 'faraday', '> 1.0.0'
```

Comment:
shouldn't that be `'~> 1.0.0'` to not accidentally update to `1.1.X` when it eventually is released in case of breaking changes?

Reply:
@sideshowcoder Done: https://github.com/yammer/circuitbox/pull/142/commits/0ff7ed0e66a3e3786306a987b4b942bc6d8fef98
Thanks.

Reply:
@yarmand If I could give my opinion about this concern, the safer way is to put two boundaries as this example:
```rb
spec.add_development_dependency 'faraday', ['>= 1.0.0', '< 2.0']
```

As is suggested in [rubygems guide patterns](https://guides.rubygems.org/patterns/)

<img width="662" alt="Screen Shot 2020-05-22 at 11 06 03" src="https://user-images.githubusercontent.com/173159/82676166-4f334980-9c1c-11ea-8c55-8343346a0a4f.png">

Reply:
I can definitely agree with this. Loose constraints on minor/patch but strong on major version.